### DISCUSSION TITLE
NOW() returns 1 hr ago [Feb 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5214] [Discord Threads]
> 🙋‍♂️ **Hirslay**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### does you browser and your database hv different timezone?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1065588734692687922)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
